src/patternfly/components/Login/docs/code.md

```diff
@@ -6,17 +6,19 @@
 | `.pf-c-login__container` | `<div>` |  Positions the login component content. |
```

Comment:
In other components, we add **Required** to all of the structural elements required to make the component, too. So `.pf-c-login`, `.pf-c-login__container`, `.pf-c-login__header`, etc